rmf_site_editor/src/workcell/save.rs

```diff
@@ -314,23 +302,71 @@ pub fn save_workcell(world: &mut World) {
             }
         };
 
+        let path = save_event.to_file;
         match save_event.format {
-            ExportFormat::Default => match workcell.to_writer(f) {
-                Ok(()) => {
-                    info!("Save successful");
+            ExportFormat::Default => {
+                info!(
+                    "Saving to {}",
+                    path.to_str().unwrap_or("<failed to render??>")
+                );
+                let f = match std::fs::File::create(path) {
+                    Ok(f) => f,
+                    Err(err) => {
+                        error!("Unable to save file: {err}");
+                        continue;
+                    }
+                };
+                match workcell.to_writer(f) {
+                    Ok(()) => {
+                        info!("Save successful");
+                    }
+                    Err(err) => {
+                        error!("Save failed: {err}");
+                    }
                 }
-                Err(err) => {
-                    error!("Save failed: {err}");
-                }
-            },
-            ExportFormat::Urdf => match workcell.to_urdf_writer(f) {
-                Ok(()) => {
-                    info!("Save successful");
-                }
-                Err(err) => {
-                    error!("Save failed: {err}");
-                }
-            },
+            }
+            ExportFormat::Urdf => {
+                match export_package(&path, &workcell) {
+                    Ok(()) => {
+                        info!("Successfully exported package");
+                    }
+                    Err(err) => {
+                        error!("Failed to export package: {err}");
+                    }
+                };
+            }
         }
     }
 }
+
+fn export_package(path: &SysPath, workcell: &Workcell) -> Result<(), Box<dyn std::error::Error>> {
+    let output_directory = path
+        .parent()
+        .ok_or("Not able to get parent")?
+        .to_str()
+        .ok_or("Invalid output directory")?
+        .to_string();
+    let package_name = path
+        .file_stem()
+        .ok_or("Not able to get file_stem")?
+        .to_str()
+        .ok_or("Invalid file name")?
+        .to_string();
+
+    let package_context = PackageContext {
+        license: "TODO".to_string(),
+        maintainers: vec![Person {
+            name: "TODO".to_string(),
+            email: "TODO".to_string(),
+        }],
```

Comment:
Imho it would be nice if maintainer and version were prepopulated to something that at least makes the package compile (i.e. version 0.0.1, some placeholder name and email) so it's a bit easier to iterate on packages. Currently since these fields are not editable by the user testing a new package requires opening the `package.xml` and updating these values to make it buildable

Reply:
Should be fixed in ee15eac.